components/Navbar.tsx

```diff
@@ -38,32 +55,57 @@ const Navbar = () => {
           {navbarLinks.map((button) => (
             <Link href={button.path} key={button.label}>
               <p
-                className={`text-sm ${
+                className={`text-small ${
                   pathname === button.path
-                    ? 'font-semibold text-primaryLight'
+                    ? 'font-semibold text-primaryLight dark:text-primaryDark'
                     : 'text-white500 dark:text-white800'
                 }`}
               >
                 {button.label}
               </p>
             </Link>
           ))}
-          {/* Temporary href below so that the link works */}
-          <Link href='/' className='flex'>
-            <Image src={downloadLight} height={20} width={20} alt='download' />
-            <p className='text-sm text-black200 dark:text-white900'>CV</p>
-          </Link>
+          {/* Download CV link */}
+          <a
+            href='/James Drysdale CV.pdf'
+            download='James Drysdale CV.pdf'
+            className='flex'
+          >
+            <Image
+              src={theme === 'light' ? downloadLight : downloadDark}
+              height={20}
+              width={20}
+              alt="download James Drysdale's CV"
+            />
+            <p className='text-small font-normal text-black200 dark:text-white900'>
+              CV
+            </p>
+          </a>
           <div className='h-6 border-l border-white500' />
           <Image
-            src={themeLight}
+            src={theme === 'light' ? themeLight : themeDark}
             height={20}
             width={20}
-            alt='light mode symbol'
+            alt='light and dark mode toggle'
             className='cursor-pointer'
-            onClick={() => {}}
+            onClick={() => setTheme(currentTheme === 'dark' ? 'light' : 'dark')}
           />
         </div>
       </section>
+      {showMobileNavbar && (
+        <div
+          className='fixed z-40 flex h-screen w-screen justify-center bg-black/30'
```

Comment:
Did you mean to use `bg-black300` instead of `bg-black/30`?

Reply:
Hey @cdsalty just chiming in here , but setting a / property in a color in tailwind is specifying an opacity layer . So by saying bg-black/30 , it's a shorthand property to say set the background color to black with 30% opacity

Reply:
Yes, I did, great spot. :)